modules/mono/editor/GodotTools/GodotTools/GodotSharpEditor.cs

```diff
@@ -661,7 +688,18 @@ protected override void Dispose(bool disposing)
                     _exportPluginWeak.Dispose();
                 }
 
+                if (IsInstanceValid(_inspectorPluginWeak))
+                {
+                    (_inspectorPluginWeak.GetRef().AsGodotObject() as InspectorPlugin)?.Dispose();
+
+                    _inspectorPluginWeak.Dispose();
+                }
+
                 GodotIdeManager?.Dispose();
+
+                // Custom signals aren't automatically currently.
```

Comment:
It's probably not a big deal, no. Now that I think about it, if we disconnect, it'd make more sense in `_DisablePlugin` since they're connected in `_EnablePlugin`.

Reply:
That makes sense to me. Although in practice the plugin is never disabled.